README.md

```diff
@@ -53,14 +66,22 @@ The `multilevel_analysis.py` script runs the model from R (currently just to sho
 
   1. Set up and activate a conda environment as above.
   2. Install R (e.g. `brew install R`)
-  4. Install MLwiN and mlnscript, for which you will need a license:
+  3. Install MLwiN and mlnscript, for which you will need a license:
     1. Sign up for an academic account at https://www.cmm.bristol.ac.uk/clients/reqform/
     2. Download `mlnscript` for MacOS/linux by filling in form at https://www.cmm.bristol.ac.uk/clients/softwaredownload/
     3. Run the installer (.dmg, .rpm, etc). If prompted for a directory, save the files to a folder such as `/opt/mln`.
     4. If the installer extracts the files to a path other than `/opt/mln`, set an environment variable `MLNSCRIPT_PATH` to where the file `mlnscript` has been saved.
+  4. Build the build the `classroommlm` R package and copy the output to the `renv` local packages dir:
 
-### Running
+    ```bash
```

Comment:
Small thing, but I'm not sure this is rendering correctly in the README.

Reply:
It was indented too much apparently. Atom's markdown-preview was more forgiving than github's markdown renderer! Should be fixed now.

Reply:
Perfect!